chess/svg.py

```diff
@@ -259,21 +262,21 @@ def board(board=None, squares=None, flipped=False, coordinates=True, lastmove=No
                 "stroke-width": str(SQUARE_SIZE * 0.2),
                 "opacity": "0.5",
                 "stroke-linecap": "butt",
-                "class": "arrow",
+                "class": "tail",
```

Comment:
Head and tail intentionally have the same CSS class, so they can be referred to together. `line.arrow` is an easy way to select the shaft.